tests/nativeapp/__snapshots__/test_commands.ambr

```diff
@@ -337,8 +336,7 @@
 # ---
 # name: test_init_no_template_success[with_inner_quotes-init_args23]
   '''
-  Native Apps project "demo""na""_""""project" has been created at: 
-  "demo""na""_""""project"
+  Native Apps project "demo""na""_""""project" has been created at: "demo""na""_""""project"
```

Comment:
Is this amount of `"` expected here?

Reply:
It is, this test verifies that we handle double quote escape sequences correctly. See https://docs.snowflake.com/en/sql-reference/identifiers-syntax for the full details on quoted identifiers and their escape sequences.